flow/record/packer.py

```diff
@@ -28,6 +28,10 @@ def identifier_to_str(identifier):
         return to_str(identifier)
 
 
+class UnknownRecordDescriptorIdentifier(Exception):
```

Comment:
```suggestion
class RecordDescriptorNotFound(Exception):
```

I think this Exception name is a bit more readable.